apps/src/lib/tools/jsdebugger/JsDebugger.story.jsx

```diff
@@ -2,62 +2,99 @@ import React from 'react';
+    ...commonReducers,
+    ...applabReducers,
+  }));
+}
 
 export default storybook => {
   const storyTable = [];
 
   const storybookStyle = {
-    position: 'relative'
+    position: 'relative',
+    width: '100%',
   };
 
   storyTable.push(
     {
       name: 'empty',
       story: () => (
-        <UnconnectedJsDebugger style={storybookStyle}/>
-      )
-    });
-
-  storyTable.push(
-    {
-      name: 'empty paused',
-      story: () => (
-        <UnconnectedJsDebugger style={storybookStyle} isDebuggerPaused/>
+        <Provider store={createApplabStore()}>
+          <JsDebugger
+            style={storybookStyle}
+          />
+        </Provider>
       )
     });
 
+  const withDebugConsoleStore = createApplabStore();
+  withDebugConsoleStore.dispatch(setPageConstants({
+    showDebugConsole: true,
+  }));
   storyTable.push(
-    {
-      name: 'with debug buttons',
-      story: () => (
-        <UnconnectedJsDebugger style={storybookStyle} debugButtons/>
-      )
-    });
+      {
+        name: 'with only debug console',
+        story: () => (
+            <div style={{height: 200}}>
+              <Provider store={withDebugConsoleStore}>
+                <JsDebugger style={storybookStyle}/>
+              </Provider>
+            </div>
+        )
+      });
 
+  const withDebugSliderStore = createApplabStore();
+  withDebugSliderStore.dispatch(setPageConstants({
+    showDebugConsole: true,
+    showDebugSlider: true,
+  }));
   storyTable.push(
-    {
-      name: 'with debug console',
-      story: () => (
-        <div style={{height: 200}}>
-          <UnconnectedJsDebugger style={storybookStyle} debugConsole/>
-        </div>
-      )
-    });
+      {
+        name: 'with speed slider',
+        story: () => (
+            <Provider store={withDebugSliderStore}>
+              <JsDebugger style={storybookStyle}/>
+            </Provider>
+        )
+      });
 
-  const emptyStore = createStore(combineReducers(commonReducers));
+  const withDebugButtonsStore = createApplabStore();
+  withDebugButtonsStore.dispatch(setPageConstants({
+    showDebugConsole: true,
+    showDebugButtons: true,
+  }));
   storyTable.push(
     {
-      name: 'connected to redux stores with nothing enabled',
+      name: 'with debug buttons',
       story: () => (
-        <Provider store={emptyStore}>
+        <Provider store={withDebugButtonsStore}>
           <JsDebugger style={storybookStyle}/>
         </Provider>
       )
     });
 
-  const showAllStore = createStore(combineReducers(commonReducers));
+  const withDebugWatchStore = createApplabStore();
+  withDebugWatchStore.dispatch(setPageConstants({
+    showDebugConsole: true,
+    showDebugWatch: true,
+  }));
+  storyTable.push(
+      {
+        name: 'with debug watch',
+        story: () => (
+            <Provider store={withDebugWatchStore}>
+              <JsDebugger style={storybookStyle}/>
+            </Provider>
+        )
+      });
```

Comment:
For all of these, would it make sense to do store creation within the story method? i.e.
```js
story: () => {
  const store = createApplabStore();
  store.dispatch(setPageConstants({}));
  return (
    <Provider store={store}>
      <JsDebugger/>
    </Provider>
  );
}